cmd/provider/cmd/testBreakingChanges/main.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2022 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Println("----Target successful,this file inside cmd/testBreakingChanges runs successfully----")
```

Comment:
This is a great start! I think as the next step, we can print the YAML files in `package/crds` folder that have a diff with the branch they are compared to. Once that's in place, we can recursively iterate and check which fields are missing in the new one.

Reply:
Yes!